### ISSUE TITLE
Proposed Ethical Web Principle: Avoid enabling and amplifying dark patterns

### ISSUE THREAD

**Issue by tantek** [w3ctag/ethical-web-principles#25]
The [W3C TAG Ethical Web Principles](https://www.w3.org/2001/tag/doc/ethical-web-principles/) mentions [enhancing individual control and power](https://www.w3.org/2001/tag/doc/ethical-web-principles/#control) and recognizes a few misbehaviors, yet focuses on decentralization, minimizing single points of failure, and enabling individual &amp; DIY developers. All of that is good, however there is the larger class of “dark pattern” harms to be named and explicitly avoided in specification and technology designs.

The Principles should explicitly note (either adding to or splitting off from “The web must enhance individuals' control and power”) the existence of “dark patterns” in web user interfaces (see [WP: Dark pattern](https://en.wikipedia.org/wiki/Dark_patterns) and [darkpatterns.org](https://www.darkpatterns.org/) for examples), with a statement similar to countering misinformation like:

We will avoid introducing technologies that create new or disproportionately enable, benefit, or amplify existing user interface dark patterns, such as confirmshaming, misdirection, friend spam, permissions pressuring or escalation, threat of data loss etc. We should also avoid new technologies that could be easily abused by existing dark patterns to more easily cause new or worse harms to users. We should design specifications that explicitly plan for and mitigate potential dark pattern abuses.

And cite either or both of those above two references. See also [twitter.com/darkpatterns](https://twitter.com/darkpatterns) for many more real world web examples.

We obviously won’t be able to prevent all dark patterns and their harms, but we can at least reduce some of them by calling them out, and avoiding new technologies that would increase the chance of users being harmed by existing and new dark patterns.

(Originally published at: https://tantek.com/2020/066/b2/avoid-enabling-amplifying-dark-patterns)

**Comment by alice** [comment]
Could you give some examples of API design which explicitly encourages or mitigates against one or more of these patterns?

**Comment by mnot** [comment]
'The web must enhance individuals' control and power' addresses this somewhat, but then conflates it with centralisation-related issues. 

I'd suggest that that item be broken into two -- one about dark patterns, and one about centralisation.

**Comment by masinter** [comment]
I thought the considerations raised by Eliot Lear in https://mailarchive.ietf.org/arch/msg/ietf/k99uQa0Eyi3sueznaw9D7XChzF0/#
 might find his issue 2 as a better formulation than "dark" (as compared to "light").

**Comment by torgo** [comment]
We discussed today. @rhiaro will work on some text on dark patterns for the intro. @mnot I don't agree that we are conflating two unrelated topics in "control and power." We're trying to make a statement that decentralised systems play a role in balancing individuals' control and power with that of large entities...? Should we try to make that more clear?

**Comment by mnot** [comment]
@torgo decentralisation is indeed about balancing individuals' control and power with that of large entities. I'm saying that it's worthy of separating that point from a discussion of dark patterns, because the latter aren't sensitive to the size of the entity abusing the user -- small web sites can use dark patterns too.

**Comment by LeaVerou** [comment]
Apparently "dark patterns" is considered inappropriate, as it associates color with malice. One politically correct term could be "manipulative design" [[1]](https://www.ccianet.org/wp-content/uploads/2021/06/2021-05-28-CCIA-Comments-on-FTC-Dark-Patterns-Workshop.pdf)

**Comment by karger** [comment]
Although a problem with "manipulative design" is that all designs manipulate (user attention, for example).  The question is the whether the intent of the manipulation is to benefit or harm the user....

**Comment by rhiaro** [comment]
Thanks @LeaVerou for mentioning that, I've done another PR to replace "dark" with "harmful" (but kept the wikipedia link the same) after also reading [this](https://dscout.com/people-nerds/vidhika-bansal)

**Comment by tantek** [comment]
tl;dr: prefer using phrase “manipulative design” instead of “harmful patterns” as a replacement for “dark patterns”.

Also adding thanks to @LeaVerou, and agreed with the reasoning in [https://www.ccianet.org/wp-content/uploads/2021/06/2021-05-28-CCIA-Comments-on-FTC-Dark-Patterns-Workshop.pdf](https://www.ccianet.org/wp-content/uploads/2021/06/2021-05-28-CCIA-Comments-on-FTC-Dark-Patterns-Workshop.pdf) (since this issue was filed!) particularly from an inclusiveness perspective, “[avoiding] connection of “dark-light dualism” to judgments about morality”.

And as @harrybr is quoted: “during the Workshop by Professor Harry Brignull, while the term “dark pattern” (originated and popularized in part by Brignull) has been useful for marketing and raising awareness, it would be both “vague and sloppy” if used as a legal term or standard.”

That being said, “harmful patterns” is vague and dilutes the intent of this issue.

It’s also not strictly accurate from the perspective of a company implementing such patterns, from their (typically attention/growth/profit-maximizing) perspective, such patterns are helpful to them personally, their shareholders/investors if any, and not harmful.

Such patterns are also too easy for such companies to hide behind a vague "intent" of supposedly "helping" people connect (quiet importing of contacts, suggesting joining radicalizing groups), or "helping" people get products they want/"need" (deceptive marketing, creating fear/insecurity in people’s minds then selling them a product to cope), etc.

More precise (and preserving of intent, meaning, and literal effect) would be the phrase “manipulative design”, also used &amp; recommended by that ccianet workshop (same link): “Rather than employ the terminology of “dark patterns,” these comments will refer to “manipulative design” practices or interfaces”.

(Originally published at: https://tantek.com/2021/278/t2/manipulative-design-instead-of-dark-patterns)

**Comment by LeaVerou** [comment]
But what about @karger's point wrt the term "manipulative design"? A lot of UX is about manipulation for a good cause, the problem here is manipulation for a harmful cause.

**Comment by harrybr** [comment]
Hi folks

I suspect that my reply wont get sent back to the mailing list since I'm
not a member (feel free to repost my reply).

The term Dark Patterns was the title of a conference talk and had I known
it was going to get used widely I would have been more thoughtful about it.
It was intended to be a bit lighthearted and reference star wars and harry
potter (plus a passing reference to other popular terms like "Dark money").

In my work as an expert witness, I usually talk about *manipulative*
*and* *deceptive
design practices.* Some designs are manipulative without being deceptive
(like confirm shaming). Deceptive designs are usually worse (since the user
doesn't know what's happening). I also try to focus on the "outcome of
harm".

As Lea said, most designed things are manipulative in some way. To add to
that, human language is always imprecise. If you want a concise term, you
can't expect much precision.

Thank you Tantek for cc:ing me.

- Harry

**Comment by rhiaro** [comment]
> It’s also not strictly accurate from the perspective of a company implementing such patterns

in that case, neither is "dark" where dark was used as a [flawed] metaphor for "bad".

> In my work as an expert witness, I usually talk about *manipulative* *and* *deceptive design practices.* 

Since the text already says "manipulate and deceive people", what if i just change the bit wrapped in the wikipedia link to "patterns" and leave it at that? The paragraph would be:

> We recognize that web technologies can be used to manipulate and deceive people, complicate isolation, and encourage addictive behaviors. We seek to mitigate against these potential abuses and <a href="https://en.wikipedia.org/wiki/Dark_pattern">patterns</a> when creating new technologies and platforms, and avoid introducing technologies that increase the chance of people being harmed in this way.

We're not coining a term here, just trying to set some pointers for further reading. Does that work?
